docs/operators_and_staking/operators.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Operators guide
+sidebar_position: 2
+description: Operators guide
+keywords:
+- Operator
+- Guide
+---
+
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Link from '@docusaurus/Link';
+import styles from '@site/src/pages/index.module.css';
+
+
+:::note
+Currently, the domain chain does not support syncing from other operator nodes; it needs to be deterministically derived from the consensus chain block by block.
+:::
+
+### Check the list of the available domains:
+In order to participate in block production, operator needs to register on a specific domain. 
+
+:::note
+Any account with the **minimum operator stake** can become an operator. 
+:::
+
+To check the list of available domains: 
+1. Proceed to [PolkadotJS](https://polkadot.js.org/apps/#/explorer)
+2. Make sure to select the correct network at the top-left corner.
+3. Go to Developer -> Chain state
+ ![Staking-1](/img/doc-imgs/operators-staking/Staking-1.png)
+4. Select `domains` under `selected state query` and choose `domainRegistry`
+5. Exclude `option`
+6. Click on `+` to query the chain state.
+ ![Staking-2](/img/doc-imgs/operators-staking/Staking-2.png)
+7. Review the list of available domains
+ ![Staking-3](/img/doc-imgs/operators-staking/Staking-3.png)
+:::tip
+In the example above the number 3 corresponds to the domainID. 
+:::
+
+### Register an operator on domain
+
+#### Create operator key:
+An operator needs a key pair to participate in bundle production.
+You can create a key using the following command:
+
+```bash
+target/production/subspace-node key generate --scheme sr25519
+```
+
+ ![Staking-4](/img/doc-imgs/operators-staking/Staking-4.png)
+
+Back up the key. Take the `public key (hex)` of the Keypair. The public key is part of the operator config we will be using later on [PolkadotJS portal](https://polkadot.js.org/apps/#/explorer).
+
+#### Insert key to Keystore:
+The key generated above needs to be added to the Keystore so that the operator node can use it to participate in bundle production.
+
+To insert the key, use the following command:
+
+```bash
+target/production/subspace-node key insert \
+--suri "<Secret phrase>" --key-type oper --scheme sr25519 --keystore-path /tmp/keystore
+```
+
+The command above assumes `/tmp/keystore` as the keystore location.
+`suri` is the secret phrase of the operator key.
+
+To register an operator on the domain: 
+1. Proceed to [PolkadotJS](https://polkadot.js.org/apps/#/explorer)
+2. Make sure to select the correct network at the top-left corner. 
+3. Select the account you want to use in `using the selected account`.
+4. Select `domains` under `submit the following extrinsic` and choose `registerOperator(domainID, amount, config)` in the dropdown.
+5. Enter the desired staking amount in the `amount` field.
+
+:::note
+In the example below, 1 TSSC is selected for staking. 18 zeros are added because of the `u128` type
```

Comment:
Unfortunately PolkadotJS interface doesn't support copying and pasting, but let's put in nevertheless in hope that it would be possible in our interface!

Reply:
Ah, thanks for clarifying Emil.